### PR TITLE
Add github actions CI

### PR DESCRIPTION
**Pull request by jacek-lewandowski**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - uses: actions/checkout@v4
+
+      - name: Create virtual environment
+        run: python -m venv venv
+
+      - name: Activate virtual environment
+        run: source venv/bin/activate
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt && pip install -r tests/requirements.txt
```
Comment (Mmuzaf) [inline .github/workflows/main.yml:25]: https://github.com/riptano/ccm/blob/master/.travis.yml#L3

Travis has different versions, should we use different ones as well?

Comment (jacek-lewandowski) [reply .github/workflows/main.yml:25]: where do we run Travis? GitHub actions uses the newest I think

Comment (michaelsembwever) [reply .github/workflows/main.yml:25]: we should converged, or remove, the travis stuff

**Review by michaelsembwever — APPROVED** [review]
optimally the travis config should be brought into line to match, but is optional IMHO as part of this PR

**Comment by jacek-lewandowski** [comment]
I have removed Travis configuration. It is not used in this repository.

**Pull request merged**